### PR TITLE
Enhance open_online action

### PR DESCRIPTION
**Pull request by kyoh86**
This pull-request will resolve:

- `open_online` action does not support ssh:// remote url. (#21)
- `open_online` action does not support WSL. (#24)

@Conni2461 Review and merge it plz.

### CODE REVIEW

File: lua/telescope/_extensions/packer.lua [hunk 1]
```diff
@@ -30,6 +30,29 @@ local setup = function(opts)
...
+    return 'xdg-open'
+  else
+    return nil
+  end
+end
+
+local function get_url(path)
+  local url = vim.fn.trim(vim.fn.system(string.format("git -C %s ls-remote --get-url", path)))
+  if string.sub(url, 1, 6) == 'ssh://' then
+    url, _ = string.gsub(url, [[^ssh://[^/]+/]], 'https://github.com/')
```
Comment (Conni2461) [inline lua/telescope/_extensions/packer.lua]: not every `ssh://` points to github.com, we can actually get the real url from that

Comment (Conni2461) [reply lua/telescope/_extensions/packer.lua]: e.g. if we have

```
local url = "ssh://git@gitlab.com:gitlab-org/gitlab.git"
url = string.gsub(url, [[^ssh://[^/]+/]], 'https://github.com/')
print(url)
```

i would expect `https://gitlab.com/gitlab-org/gitlab` but i currently get `https://github.com/gitlab.git` which is completely wrong

Comment (kyoh86) [reply lua/telescope/_extensions/packer.lua]: I see.
I'll fix it

Comment (kyoh86) [reply lua/telescope/_extensions/packer.lua]: done.

File: lua/telescope/_extensions/packer.lua [hunk 2]
```diff
@@ -30,6 +30,30 @@ local setup = function(opts)
...
+    return nil
+  end
+end
+
+local function get_url(path)
+  local url = vim.fn.trim(vim.fn.system(string.format("git -C %s ls-remote --get-url", path)))
+  if string.sub(url, 1, 6) == 'ssh://' then
+    url, _ = string.gsub(url, '^ssh://%w+@', 'https://')
+    url, _ = string.gsub(url, '^ssh://', 'https://')
+  end
```
Comment (Conni2461) [inline lua/telescope/_extensions/packer.lua:53]: oh one more thing it could also start with `git@github.com:nvim-telescope/telescope.nvim.git` e.g.

**Pull request closed without merging**